higlass/_version.py

```diff
@@ -518,3 +519,24 @@ def get_versions():
     return {"version": "0+unknown", "full-revisionid": None,
             "dirty": None,
             "error": "unable to compute version", "date": None}
+
+
+def get_js_version():
```

Comment:
`_version.py` was generated by versioneer, so it might not be safe to add this here if it gets updated in the future. Maybe it should go in `__init__.py`.

Reply:
Or directly in the widgets module.

Reply:
We can get rid of versioneer, and just add a version number here. Versioneer seems like a very complex solution to a simple problem. What do you think?

Reply:
I am not a python expert but I like this idea. versioneer seems cool but it makes things intransparent as the version number is not visible in the code and adds a lot of complexity.

Do you guys have a simple best practice example?